src/restful/controllers/reviewCycleControllers.js

```diff
@@ -0,0 +1,176 @@
+import Response from "../../system/helpers/Response";
+import ReviewCycleService from "../../services/reviewCycleServices";
+
+export default class ReviewCycleControllors {
+  static async create(req, res) {
+    try {
+      const { startAt, endingAt } = req.body;
+      ReviewCycleService.create({
+        startAt,
+        endingAt,
+      })
+        .then((reviewCycle) => {
+          return Response.success(res, 200, {
+            message: "reviewCycle created successfully",
+            data: reviewCycle,
+          });
+        })
+        .catch((error) => {
+          return Response.error(res, 403, {
+            message: "reviewCycle failed to be created",
+            error: error.message,
+          });
+        });
+    } catch (error) {
+      return Response.error(res, 500, {
+        message: "server error",
+        error: error.message,
+      });
+    }
+  }
+
+  static async getALl(req, res) {
+    try {
+      ReviewCycleService.findAll()
+        .then((reviewCycle) => {
+          return Response.success(res, 200, {
+            message: "reviewCycles retreived successfully",
+            data: reviewCycle,
+          });
+        })
+        .catch((error) => {
+          return Response.error(res, 403, {
+            message: "reviewCycles failed to be retreived",
+            error: error.message,
+          });
+        });
+    } catch (error) {
+      return Response.error(res, 500, {
+        message: "server error",
+        error: error.message,
+      });
+    }
+  }
+
+  static async getOne(req, res) {
+    try {
+      const { id } = req.params;
+      ReviewCycleService.findById(id)
+        .then((reviewCycle) => {
+          return Response.success(res, 200, {
+            message: "reviewCycle retreived successfully",
+            data: reviewCycle,
+          });
+        })
+        .catch((error) => {
+          return Response.error(res, 403, {
+            message: "reviewCycle failed to be retreived",
+            error: error.message,
+          });
+        });
+    } catch (error) {
+      return Response.error(res, 500, {
+        message: "server error",
+        error: error.message,
+      });
+    }
+  }
+
+  static async start(req, res) {
+    try {
+      const { id } = req.params;
+      const reviewCycle = await ReviewCycleService.findONE({ id });
+      if (!reviewCycle) {
+        return Response.error(res, 404, {
+          message: "Unable to find this review cycle",
+        });
+      }
+      reviewCycle.set("started", true);
+      await reviewCycle.save();
```

Comment:
I think it would be better if we did like this on `create, getAll, delete, etc controllers` instead of using `.then().catch()`
To keep code consistency.